src/TouristInfoCards.jsx

```diff
@@ -0,0 +1,18 @@
+const TouristInfoCards = props => (
```

Comment:
This only returns one `Card`, it is good to be as clear as possible what a component is going to return, so I would suggest losing the S on the end of it.